website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -108,6 +108,7 @@ apply to both Consul Enterprise and Consul community edition (CE).
 
 | Consul Version | Default `consul-dataplane` Version   | Other compatible `consul-dataplane` Versions |
 | -------------- | -------------------------------------|----------------------------------------------|
+| 1.20.x CE      | 1.6.x (Envoy 1.31.x)                 | 1.5.x (Envoy 1.29.x)                         |
 | 1.19.x CE      | 1.5.x (Envoy 1.29.x)                 | 1.4.x (Envoy 1.28.x)                         |
```

Comment:
Interestingly, 1.20.x includes Envoy 1.31.x ([link](https://github.com/hashicorp/consul/blob/release/1.20.x/envoyextensions/xdscommon/ENVOY_VERSIONS)) while 1.19.x never made it to 1.30.x and stayed at 1.29.x ([link](https://github.com/hashicorp/consul/blob/release/1.19.x/envoyextensions/xdscommon/ENVOY_VERSIONS)), but this table reflects current reality

Reply:
We had to jump two versions at some point bc we'd started to lag >1 release on latest Envoy vs. latest Consul due to release cadence differences. 1.20 happened to be selected as the version for the jump.